### ISSUE TITLE
Service fails to start - WARNING: /etc/nginx/dhparam/dhparam

### ISSUE THREAD

**Issue by websecusa** [nginx-proxy/nginx-proxy#1232]
Hi,
Sematext has published an article referencing nginx-proxy
https://sematext.com/blog/docker-elasticsearch-swarm/

They provided the below:
docker service create --mode global \
--name proxy -p 80:80 \
--network elasticsearch-frontend \
--network elasticsearch-backend  \
--mount type=bind,src=/var/run/docker.sock,target=/tmp/docker.sock:ro \
jwilder/nginx-proxy

Unfortunately this fails to run, with the below errors.

proxy.0.sj6fv33fh1zg@INDOCKERLN03P    | ERROR: you need to share your Docker host socket with a volume at /tmp/docker.sock
proxy.0.sj6fv33fh1zg@INDOCKERLN03P    | Typically you should run your jwilder/nginx-proxy with: `-v /var/run/docker.sock:/tmp/docker.sock:ro`
proxy.0.sj6fv33fh1zg@INDOCKERLN03P    | See the documentation at http://git.io/vZaGJ
proxy.0.sj6fv33fh1zg@INDOCKERLN03P    | WARNING: /etc/nginx/dhparam/dhparam.pem was not found. A pre-generated dhparam.pem will be used for now while a new one
proxy.0.sj6fv33fh1zg@INDOCKERLN03P    | is being generated in the background.  Once the new dhparam.pem is in place, nginx will be reloaded.

**Comment by maxguru** [comment]
I'm getting the following error (along with the above warning regarding dhparam) and container exits,
```
2019/02/09 21:23:59 [emerg] 34#34: PEM_read_bio_DHparams("/etc/nginx/dhparam/dhparam.pem") failed (SSL: error:0906D06C:PEM routines:PEM_read_bio:no start line:Expecting: DH PARAMETERS)
```

**Comment by statte** [comment]
Same for me

**Comment by maxguru** [comment]
I think this is a duplicate of #1226.

**Comment by martijnvwezel** [comment]
This should be closed, because duplicated.

**Comment by buchdag** [comment]
like #1226, fixed by #1213
